src/fork_gc.c

```diff
@@ -63,23 +63,30 @@ static char *ForkGc_FDReadBuffer(int fd, size_t *len) {
     return NULL;
   }
   char *buff = rm_malloc(*len * sizeof(char));
-  read(fd, buff, *len);
+  ssize_t l = read(fd, buff, *len);
+  assert(l == *len);
```

Comment:
no if the child fails the master will not crash and I think its better to know about it then ignore it.

Reply:
If the child fails then the parent will crash because the assertion will fail (i.e. read will not return `*len`). It's better to print to screen or log.